src/main/java/oshi/hardware/stores/HWNetworkStore.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2016 com.github.dblock.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * enrico[dot]bianchi[at]gmail[dot]com
+ *    com.github.dblock - initial API and implementation and/or initial documentation
+ */
+package oshi.hardware.stores;
+
+import javax.json.Json;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+import oshi.json.NullAwareJsonObjectBuilder;
+import oshi.json.OshiJsonObject;
+
+/**
+ * Store object of network interfaces attributes
+ *
+ * @author enrico[dot]bianchi[at]gmail[dot]com
+ */
+public class HWNetworkStore implements OshiJsonObject {
```

Comment:
That makes sense, but we've named all the first-level interfaces and objects corresponding to physical names. CentralProcessor, GlobalMemory, PowerSource, Sensors, Display, FileStore, and DiskStore (where Store = Storage).  Network interfaces (and Java `interface`s) are broken if they store anything. They're just supposed to send stuff from one place to another. :)

Since Java has already claimed the `NetworkInterface` name, we should modify it slightly.  Mac and Windows methods make a lot of use of the IF abbreviation (so dos Linux `ifconfig`) so we should definitely use IF or Interface in the name. NWInterface or NetworkIF or HWNetworkInterface or HWNetInterface or NetIF...